testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -1680,13 +1592,15 @@ def test_dihedral_equals_improper(self, universe):
 
     def test_bond(self, universe):
         universe.trajectory.rewind()  # just to make sure...
```

Comment:
While we're tidying, get rid of these rewinds

Reply:
Done.